earthpy/spatial.py

```diff
@@ -319,19 +319,20 @@ def colorbar(mapobj, size="3%", pad=0.09, aspect=20):
 # Function to plot all layers in a stack
 def plot_bands(arr, title=None, cmap="Greys_r",
                figsize=(12, 12), cols=3, extent=None):
-    """Plot each layer in a raster stack converted into a numpy array for
-       quick visualization.
+    """Plot each layer in a raster stack read from rasterio in
+    (band, row ,col) order as a numpy array. plot_bands will create an
+    individual plot for each band in a grid.
 
     Parameters
     ----------
     arr: numpy array
-        An n-dimensional numpy array
+        An n-dimensional numpy array in (band, row, col) order
```

Comment:
i think i'd like to keep this as is a band, row, col as a convention for raster data. and i think rasterio too. n confuses the vars a bit more.

Reply:
Cool - would you be comfortable with changing "in (band, row, col) order" to "with shape (band, row, col)", to better match numpy array terminology? I'm also thinking this might avoid confusing users, because "order" is also used to talk about the order of 2d arrays (e.g., row vs. column major order).

Reply:
Rasterio uses "shape" instead of "order" on that line, so wouldn't changing to "with shape (band, row, col)" be more consistent with rasterio than using "order"? I'm going to update my suggested change, since I'm not sure I was super clear in my last comment